pkg/bot/discord.go

```diff
@@ -181,18 +182,19 @@ func (b *Discord) Type() config.IntegrationType {
 }
 
 // TODO: Support custom routing via annotations for Discord as well
-func (b *Discord) getChannelsToNotify() []string {
-	// TODO(https://github.com/kubeshop/botkube/issues/596): Support source bindings - filter events here or at source level and pass it every time via event property?
-	var channelsToNotify []string
-	for _, channelCfg := range b.getChannels() {
-		if !channelCfg.notify {
-			b.log.Info("Skipping notification for channel %q as notifications are disabled.", channelCfg.Identifier())
-			continue
+func (b *Discord) getChannelsToNotify(eventSources []string) []string {
```

Comment:
I see multiple `getChannelsToNotify` private methods in different bot implementations, but they are nearly doing same thing. (I see there is only one exception for teams, but maybe it can also use same logic). This method seems a good candidate for extracting to util package to be consumed by each bot, wdyt?

Reply:
Agreed. It's legacy code.

This can def be improved in the future.